### ISSUE TITLE
[Crash]: Variable modifier target for net/minecraft/class_310::setScreen$1 was removed by another injector

### ISSUE THREAD

**Issue by timingplanet** [Fuzss/puzzleslib#61]
### Mod Loader (Required)

Fabric

### Minecraft Version (Required)

1.20.1

### Mod Version (Required)

v8.0.15

### Notes (Required)

Idk if its having a issue with a mod or what but it keep crashing and all I can see is that it's not happy with puzzle lib.

### Crash Report (Required)

https://pastebin.com/K9wfeisV

**Comment by timingplanet** [comment]
Note: changed mod version to PuzzlesLib-v8.0.7-1.20.1-Fabric.jar and crashes have stopped

**Comment by timingplanet** [comment]
Well it works fine with the older version and i tried removing a ton of
mods still was crashing

**Comment by ygoroffici** [comment]
> This is a mod conflict. My implementation didn't change from 8.0.7 to 8.0.15. Did you update any other mods? The only way to find this otherwise is to remove mods step by step until it works again.

I found the conflict, it's conflicting with the latest Emojiful version (4.2.0) for Fabric (I don't know if it's happening with Forge as well)
[This class is likely what's causing the conflict!](https://github.com/InnovativeOnlineIndustries/Emojiful/blob/1.19-multiloader/Fabric/src/main/java/com/hrznstudio/emojiful/mixin/fabric/ScreenEmojifulMixin.java)

**Comment by timingplanet** [comment]
Interesting ill update puzzle lib and remove emojiful and see if that fixes the issue but thats weird that they are conflicting in the newest lateast version

**Comment by timingplanet** [comment]
After testing my self it seems emojiful is the conflict

**Comment by bipsydev** [comment]
Adding to this, I got a similar error with Krypton installed about conflicting mixins on `net.minecraft.class_1493`:

```
---- Crashed! ----
Date/Time: 2023/08/08 18:42:30.7278

-- Crash --

org.quiltmc.loader.impl.FormattedException: java.lang.BootstrapMethodError: java.lang.RuntimeException: Mixin transformation of net.minecraft.class_1493 failed
	at org.quiltmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:526)
	at org.quiltmc.loader.impl.launch.knot.Knot.launch(Knot.java:82)
	at org.quiltmc.loader.impl.launch.knot.KnotClient.main(KnotClient.java:28)
	at org.prismlauncher.launcher.impl.StandardLauncher.launch(StandardLauncher.java:88)
	at org.prismlauncher.EntryPoint.listen(EntryPoint.java:126)
	at org.prismlauncher.EntryPoint.main(EntryPoint.java:71)
Caused by: java.lang.BootstrapMethodError: java.lang.RuntimeException: Mixin transformation of net.minecraft.class_1493 failed
	at net.minecraft.class_1299.<clinit>(class_1299.java:291)
	at net.minecraft.class_1802.<clinit>(class_1802.java:797)
	at net.minecraft.class_6089.method_9530(class_6089.java:56)
	at net.minecraft.class_4970$class_4971.method_26172(class_4970.java:1032)
	at net.minecraft.class_4970$class_4971.method_26218(class_4970.java:1028)
	at net.minecraft.class_4970.method_9549(class_4970.java:349)
	at net.minecraft.class_4970$class_4971$class_3752.<init>(class_4970.java:1293)
	at net.minecraft.class_4970$class_4971.method_26200(class_4970.java:843)
	at net.minecraft.class_2246.<clinit>(class_2246.java:1185)
	at net.minecraft.class_2358.method_10199(class_2358.java:326)
	at net.minecraft.class_2966.method_12851(class_2966.java:54)
	at net.minecraft.client.main.Main.main(Main.java:166)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at org.quiltmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:524)
	... 5 more
Caused by: java.lang.RuntimeException: Mixin transformation of net.minecraft.class_1493 failed
	at org.quiltmc.loader.impl.launch.knot.KnotClassDelegate.getPostMixinClassByteArray(KnotClassDelegate.java:458)
	at org.quiltmc.loader.impl.launch.knot.KnotClassDelegate.tryLoadClass(KnotClassDelegate.java:244)
	at org.quiltmc.loader.impl.launch.knot.KnotClassDelegate.loadClassOnly(KnotClassDelegate.java:164)
	at org.quiltmc.loader.impl.launch.knot.KnotClassDelegate.loadClass(KnotClassDelegate.java:150)
	at org.quiltmc.loader.impl.launch.knot.KnotClassLoader.loadClass(KnotClassLoader.java:228)
	at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:520)
	... 22 more
Caused by: org.spongepowered.asm.mixin.transformer.throwables.MixinTransformerError: An unexpected critical error was encountered
	at org.spongepowered.asm.mixin.transformer.MixinProcessor.applyMixins(MixinProcessor.java:392)
	at org.spongepowered.asm.mixin.transformer.MixinTransformer.transformClass(MixinTransformer.java:234)
	at org.spongepowered.asm.mixin.transformer.MixinTransformer.transformClassBytes(MixinTransformer.java:202)
	at org.quiltmc.loader.impl.launch.knot.KnotClassDelegate.getPostMixinClassByteArray(KnotClassDelegate.java:453)
	... 27 more
Caused by: org.spongepowered.asm.mixin.throwables.MixinApplyError: Mixin [#puzzleslib:puzzleslib.fabric.mixins.json:WolfFabricMixin from mod puzzleslib] from phase [DEFAULT] in config [#puzzleslib:puzzleslib.fabric.mixins.json] FAILED during APPLY
	at org.spongepowered.asm.mixin.transformer.MixinProcessor.handleMixinError(MixinProcessor.java:638)
	at org.spongepowered.asm.mixin.transformer.MixinProcessor.handleMixinApplyError(MixinProcessor.java:589)
	at org.spongepowered.asm.mixin.transformer.MixinProcessor.applyMixins(MixinProcessor.java:379)
	... 30 more
Caused by: org.spongepowered.asm.mixin.transformer.throwables.InvalidMixinException: Conflicting opcodes ACONST_NULL and ACONST_NULL at offset 2 in synthetic bridge method method_48926()Lnet/minecraft/class_1924;
	at org.spongepowered.asm.mixin.transformer.MixinPreProcessorStandard.attachUniqueMethod(MixinPreProcessorStandard.java:564)
	at org.spongepowered.asm.mixin.transformer.MixinPreProcessorStandard.attachMethods(MixinPreProcessorStandard.java:351)
	at org.spongepowered.asm.mixin.transformer.MixinPreProcessorStandard.attach(MixinPreProcessorStandard.java:299)
	at org.spongepowered.asm.mixin.transformer.MixinPreProcessorStandard.createContextFor(MixinPreProcessorStandard.java:277)
	at org.spongepowered.asm.mixin.transformer.MixinInfo.createContextFor(MixinInfo.java:1289)
	at org.spongepowered.asm.mixin.transformer.MixinApplicatorStandard.apply(MixinApplicatorStandard.java:294)
	at org.spongepowered.asm.mixin.transformer.TargetClassContext.apply(TargetClassContext.java:421)
	at org.spongepowered.asm.mixin.transformer.TargetClassContext.applyMixins(TargetClassContext.java:403)
	at org.spongepowered.asm.mixin.transformer.MixinProcessor.applyMixins(MixinProcessor.java:363)
	... 30 more

```

[This](https://www.reddit.com/r/fabricmc/comments/svgsbe/runtimeexception_mixin_transformation_of/) helped me narrow it down, hopefully helps here as well?

**Comment by Fuzss** [comment]
@bipsydev This is unrelated to this issue, remove the doggo mod for now, the dev needs to fix it on their end.

**Comment by Fuzss** [comment]
Fixed in latest versions of Puzzles Lib (v3.5.7 & v8.0.21).
